scannerc.go

```diff
@@ -2847,7 +2847,7 @@ func yaml_parser_scan_line_comment(parser *yaml_parser_t, token_mark yaml_mark_t
 			continue
 		}
 		if parser.buffer[parser.buffer_pos+peek] == '#' {
-			seen := parser.mark.index+peek
+			seen := parser.mark.index + peek
```

Comment:
Are these changes intended? Should we revert this file?

Reply:
I'm torn, these aren't intended. It's a result of having changes in this file and then removing them, the autoformatter still runs on save. 

I'm usually opposed to unintended changes, but it also feels wrong to intentional rebreak style. So I would argue its a beneficial change to fix style even when I'm not otherwise touching the filee